examples/example3/README.md

```diff
@@ -0,0 +1,40 @@
+## Example 3 – Using virtual resources and collection in a master-agent set-up.
+
+
+This is an (almost) complete example for a master-agent set-up using virtual resources in Puppet.
```

Comment:
Yes, I can add 1-2 more sentences on services and probably also on where to define apply rules.